net/strongswan/Makefile

```diff
@@ -459,6 +471,8 @@ define Package/strongswan/install
 	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/strongswan.conf $(1)/etc/
 	$(INSTALL_DIR) $(1)/usr/lib/ipsec
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libstrongswan.so.* $(1)/usr/lib/ipsec/
+	if [ -f $(PKG_INSTALL_DIR)/usr/lib/ipsec/libnttfft.so ]; then \
+		$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libnttfft.so.* $(1)/usr/lib/ipsec/; fi
```

Comment:
Please make a separate package for libnttfft.so instead of conditionally including it in the strongswan package, and add it as dependency to the module packages that require it. The strongswan-libtls package is a good example.